### PR TITLE
*: add br scenario demonstration doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

Add BR Scenario Demonstration document.

### What is the related PR or file link(s)? 

- [ ] Reference link(s):https://github.com/pingcap/docs/issues/1930
- [x] This PR is to align with: https://github.com/pingcap/docs-cn/pull/2167, https://github.com/pingcap/docs-cn/pull/2227, https://github.com/pingcap/docs-cn/pull/2216
- [ ] N/A (not applicable)

### Which TiDB version(s) does your changes apply to? 

- [ ] All active versions: dev, v3.0, v2.1, v3.1
- [x] dev (the latest development version)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] All active and inactive versions
- [ ] N/A (not applicable)

**Note:** If your changes apply to multiple TiDB versions, make sure you update the documents in the **corresponding** version folders such as "dev", "v3.0", "v2.1" and "v3.1" in this PR.

- [ ] Updated one version first. Will update other versions after I get two LGTMs.

### CODE REVIEW

**Comment by TomShawn** [comment]
@lilin90 @3pointer @GregoryIan PTAL, thanks!

File: TOC.md [hunk 1]
```diff
@@ -85,7 +85,8 @@
     - [Common Ansible Operations](/how-to/deploy/orchestrated/ansible-operations.md)
     + Backup and Restore
       - [Use `mydumper` and `loader`](/how-to/maintain/backup-and-restore/mydumper-loader.md)
-      - [Use BR](/how-to/maintain/backup-and-restore/br.md)
+      - [Use BR](/reference/tools/br/br.md)
+      - [BR Scenario Demonstration](/reference/tools/br/use-case.md)
```
Comment (lilin90) [inline TOC.md]:
```suggestion
      - [BR Usage Scenarios](/reference/tools/br/use-cases.md)
```

File: reference/tools/br/br.md [hunk 2]
```diff
@@ -343,6 +343,8 @@ In the above command, `--table` specifies the name of the table to be restored.
 - It is recommended that you use a storage hardware with high throughput, because the throughput of a storage hardware limits the backup and restoration speed.
 - It is recommended that you perform the backup operation during off-peak hours to minimize the impact on applications.
 
+For more recommended practices of using BR, refer to [BR Scenario Demonstration](/reference/tools/br/use-case.md).
```
Comment (lilin90) [inline reference/tools/br/br.md]:
```suggestion
For more recommended practices of using BR, refer to [BR Scenario Demonstration](/reference/tools/br/use-cases.md).
```

File: reference/tools/br/use-case.md [hunk 3]
```diff
@@ -0,0 +1,418 @@
+---
+title: BR Scenario Demonstrations
```
Comment (lilin90) [inline reference/tools/br/use-case.md]: ditto

File: reference/tools/br/use-cases.md [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+
+**Backup Errors**: the errors occurred during the backup process. No error occurs in normal situations. Even if a few errors occur, the backup operation has the retry mechanism which might increase the backup time but does not affect the operation correctness.
+
+![img](/media/br/backup-errors.png)
+
+**Checksum Request Duration**: the duration of the admin checksum request in the backup cluster.
+
+![img](/media/br/checksum-duration.png)
+
+#### Learn the backup results
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
#### Backup results explanation
```

File: reference/tools/br/use-cases.md [hunk 5]
```diff
@@ -0,0 +1,418 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+bin/br backup table --db batchmark --table order_line -s local:///br_data --pd 172.16.5.198:2379 --log-file backup-nfs.log
+```
+
+#### Monitoring metrics for the backup
+
+During the backup process, pay attention to the following metrics on the monitoring panels to learn the status of the backup process.
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
During the backup process, pay attention to the following metrics on the monitoring panels to get the status of the backup process.
```

File: reference/tools/br/use-cases.md [hunk 6]
```diff
@@ -0,0 +1,418 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+bin/br restore table --db batchmark --table order_line -s local:///br_data --pd 172.16.5.198:2379 --log-file restore-nfs.log
+```
+
+#### Monitoring metrics for the restoration
+
+During the restoration process, pay attention to the following metrics on the monitoring panels to learn the status of the restoration process.
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
During the restoration process, pay attention to the following metrics on the monitoring panels to get the status of the restoration process.
```

File: reference/tools/br/use-cases.md [hunk 7]
```diff
@@ -0,0 +1,418 @@
...
+
+**Restore Errors**: the errors occurred during the restoration process.
+
+![img](/media/br/restore-errors.png)
+
+**Checksum Request duration**: the duration of the admin checksum request. This duration for the restoration is longer than that for the backup.
+
+![img](/media/br/restore-checksum.png)
+
+#### Learn the restoration results
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
#### Restoration results explanation
```

File: reference/tools/br/use-cases.md [hunk 8]
```diff
@@ -0,0 +1,418 @@
...
+
+Execute the `br backup` command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+bin/br backup table --db batchmark --table order_line -s local:///home/tidb/backup_local/ --pd 172.16.5.198:2379 --log-file backup_local.log
+```
+
+During the backup process, pay attention to the metrics on the monitoring panels to learn the status of the backup process. See [Monitoring metrics for the backup](#monitoring-metrics-for-the-backup) for details.
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
During the backup process, pay attention to the metrics on the monitoring panels to get the status of the backup process. See [Monitoring metrics for the backup](#monitoring-metrics-for-the-backup) for details.
```

File: reference/tools/br/use-cases.md [hunk 9]
```diff
@@ -0,0 +1,418 @@
...
+
+#### Learn the backup results
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
#### Backup results explanation
```

File: reference/tools/br/use-cases.md [hunk 10]
```diff
@@ -0,0 +1,418 @@
...
+
+Execute the `br restore` command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+bin/br restore table --db batchmark --table order_line -s local:///home/tidb/backup_local/ --pd 172.16.5.198:2379 --log-file restore_local.log
+```
+
+During the restoration process, pay attention to the metrics on the monitoring panels to learn the status of the restoration process. See [Monitoring metrics for the restoration](#monitoring-metrics-for-the-restoration) for details.
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
During the restoration process, pay attention to the metrics on the monitoring panels to get the status of the restoration process. See [Monitoring metrics for the restoration](#monitoring-metrics-for-the-restoration) for details.
```

File: reference/tools/br/use-cases.md [hunk 11]
```diff
@@ -0,0 +1,418 @@
...
+
+#### Learn the restoration results
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
#### Restoration results explanation
```

File: reference/tools/br/use-cases.md [hunk 12]
```diff
@@ -0,0 +1,418 @@
...
+summary: Learn the scenarios of backing up and restoring data using BR.
+category: how-to
+---
+
+# BR Usage Scenarios
+
+[Backup & Restore](/reference/tools/br/br.md) (BR) is a command-line tool for distributed backup and restoration of the TiDB cluster data. This document describes the processes of operating BR in [four scenarios](#user-scenarios) that aims to help you achieve the following goals:
+
+* Back up and restore data using a network disk or local disk correctly.
+* Learn the status of a backup or restoration operation through monitoring metrics.
```
Comment (lilin90) [inline reference/tools/br/use-cases.md]:
```suggestion
* Get the status of a backup or restoration operation through monitoring metrics.
```

**Comment by lilin90** [comment]
@3pointer PTAL

**Pull request merged**